test/spec/ref_cast.wast

```diff
@@ -131,6 +155,7 @@
 (assert_return (invoke "test-ref-test-any") (i32.const 1))
 (assert_return (invoke "test-ref-cast-struct"))
 (assert_return (invoke "test-br-on-cast-struct") (i32.const 1))
+(assert_return (invoke "test-br-on-cast-null-struct") (i32.const 1))
```

Comment:
Two new exports, but only one new invoke of an export - is the latter missing?

Reply:
Oops, yes, good catch.